app/src/main/java/org/fossasia/susi/ai/helper/PrefManager.java

```diff
@@ -51,7 +51,7 @@ public static int getInt(String preferenceKey, int preferenceDefaultValue) {
         return getPreferences().getInt(preferenceKey, preferenceDefaultValue);
     }
 
-    public static void putTheme(String preferenceKey, String value){
+    public static void putTheme(String value){
```

Comment:
In putstring you already added key. Constant.Theme. so no need of providing key in parameter. It is specifically for theme.